models/haskell/TermZipper.hs

```diff
@@ -0,0 +1,119 @@
+module Hopper.Internal.Core.TermZipper where
+
```

Comment:
Derp. Yay. Ok. Cool.

On Wednesday, April 13, 2016, Brian Schroeder <notifications@github.com>
wrote:

> In models/haskell/TermZipper.hs
> <https://github.com/hopper-lang/hopper/pull/7#discussion_r59627960>:
>
> > @@ -0,0 +1,119 @@
> > +module Hopper.Internal.Core.TermZipper where
> > +
>
> Correct. Note the path models/haskell/TermZipper.hs
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/hopper-lang/hopper/pull/7/files/e91bbaa90485e5b423b1944934c3712b8c63808a#r59627960>
>